src/common/transformations/include/transformations/utils/utils.hpp

```diff
@@ -87,6 +87,8 @@ inline std::string get_ie_output_name(const Output<Node>& output) {
     return get_ie_output_name(ov::Output<const Node>(output.get_node(), output.get_index()));
 }
 
+float cast_eps_to_float(double eps_d);
```

Comment:
Add description how this cast work, as it is not just simple casting.

Reply:
Description has been added.